joomla/joomla-cms
# [4.0] Moving com_finder submenu to mainmenu

This adds menu item types for com_finder views, removes the submenu and adds them into the mainmenu. 

### How to test
1. Apply patch
2. Go to database view and click on "fix"
3. Go to menu manager -> All menu items and click "rebuild"
4. See that the Smart Search menu item can now open up and contains 4 sub menu items and that the submenu in the actual views has been removed.
